### PR TITLE
Main heading component

### PR DESCRIPTION
**Pull request by fares98**
- Create a MainHeading component and create the validation for the component 
- Add the main font for the project

Relates: #54

### CODE REVIEW

File: src/components/MainHeading/main-heading.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@font-face {
+  font-family: "vagroun";
+  src: url("../../assets/fonts/vagroun.ttf") format("truetype");
+}
+
+h2 {
```
Comment (Fatmasiam) [inline src/components/MainHeading/main-heading.css:6]: please don't use the `tag name` in CSS

**Review by Fatmasiam — CHANGES_REQUESTED** [review]

File: src/components/MainHeading/index.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './main-heading.css';
+
+const MainHeading = ({ text }) => {
+  return (
+    <div>
```
Comment (Fatmasiam) [inline src/components/MainHeading/index.jsx]: why you put it on the `div` i think `h2` enough

File: src/components/MainHeading/index.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './main-heading.css';
+
+const MainHeading = ({ text }) => {
```
Comment (Fatmasiam) [inline src/components/MainHeading/index.jsx]:
```jsx
const MainHeading = ({ text }) =>  <h2>{text}</h2>;
```

**Review by Fatmasiam — DISMISSED** [review]

**Review by yosefanajjar — DISMISSED** [review]

**Review by AlaaTaima — DISMISSED** [review]

File: src/components/MainHeading/main-heading.css [hunk 4]
```diff
@@ -0,0 +1,12 @@
+@font-face {
+  font-family: "vagroun";
```
Comment (alaa-yasin) [inline src/components/MainHeading/main-heading.css]: Can you change the font-family to "VAG Rounded", please?

File: src/components/MainHeading/main-heading.css [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  src: url("../../assets/fonts/vagroun.ttf") format("truetype");
+}
+
+.main-head {
+  font-weight: 600;
+  font-size: 48px;
+  text-align: left;
+  font-family: "vagroun" !important;
```
Comment (alaa-yasin) [inline src/components/MainHeading/main-heading.css]: If you do so, Don't forget to change it here as well :)

File: src/components/MainHeading/main-heading.css [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+  font-weight: normal;
```
Comment (alaa-yasin) [inline src/components/MainHeading/main-heading.css]: It is repeated

**Pull request merged**